doc/classes/TileSet.xml

```diff
@@ -143,6 +143,14 @@
 				Returns the custom data layers count.
 			</description>
 		</method>
+		<method name="get_navigation_layer_layer_value" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="layer_index" type="int" />
+			<param index="1" name="layer_number" type="int" />
+			<description>
+				Returns whether or not the specified navigation layer of the custom data layer identified by the given [param layer_index] is enabled, given a navigation_layers [param layer_number] between 1 and 32.
```

Comment:
"custom data layer" is a special term for additional data you can paint on your Tileset (like, a boolean or an int). Here I think you should use "TileSet navigation data layer" I believe. 

Sorry, I know the naming isn't great... I would have liked to find a better one ^_^"

(same below)